.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "30 1 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.'
+        stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.'
+        stale-issue-label: 'Stale'
+        stale-pr-label: 'Stale'
+        days-before-pr-stale: 90
+        days-before-issue-stale: 365
+        days-before-pr-close: -1
+        days-before-issue-close: -1
```

Comment:
Looks like this can be consolidated to `days-before-close: -1` if we want.

On the other hand, having both explicitly stated (as you have it here) can be helpful in the future if we want to tweak one of these. Either way is fine with me.

Reply:
Yes, in the future we should automatically close PRs and issues and those are just place holders in case we want to have specific times for each.